tailwind.config.ts

```diff
@@ -1,102 +1,184 @@
+				2.6: "2.6rem",
+				2.8: "2.8rem",
+				3: "3rem",
+				3.2: "3.2rem",
+				3.4: "3.4rem",
+				4.8: "4.8rem",
+				5.5: "5.5rem",
+				6.2: "6.2rem",
+				7: "7rem",
+				10: "10rem",
+				21.3: "21.3rem",
+			},
+
+			borderRadius: {
+				6: "6px",
+				8: "8px",
+				10: "10px",
+			},
+
+			fontSize: {
+				1: "1rem",
+				1.2: "1.2rem",
+				1.4: "1.4rem",
+				1.6: "1.6rem",
+				2: "2rem",
+				2.4: "2.4rem",
+			},
+
 			screens: {
-				'3xl': '2000px',
+				"3xl": "2000px",
 			},
+
 			backgroundImage: {
 				contours: "url('/assets/images/shared/bg-contours.png')",
-				'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
-				'gradient-conic':
-					'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
+				"contours-old": "url('/assets/images/shared/contours-old.png')",
+				"gradient-radial": "radial-gradient(var(--tw-gradient-stops))",
+				"gradient-conic":
+					"conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
 			},
+
 			boxShadow: {
-				'auth-layout-shadow': ' 0px 2px 32px 0px rgba(0, 0, 0, 0.08)',
+				"auth-layout-shadow": "0px 2px 32px 0px rgba(0, 0, 0, 0.08)",
 			},
+
 			colors: {
-				text: '#1B1818',
-				successText: '#484848',
-				formBtn: '#008080',
-				validationMsg: '#268384',
+				text: "#1B1818",
+				successText: "#484848",
+				placeholder: "#8D8B8B",
+				unfocused: "#A8CCCC",
+				formBtn: "#008080",
+				semiWhite: "#E6EAEE",
+				validationMsg: "#268384",
+				popover: "hsl(0 0% 100%)",
+				"popover-foreground": "hsl(222.2 47.4% 11.2%)",
+				accent: "hsl(210 40% 96.1%)",
+				"accent-foreground": "hsl(222.2 47.4% 11.2%)",
+				input: "hsl(214.3 31.8% 91.4%)",
+				ring: "hsl(215 20.2% 65.1%)",
+				muted: "hsl(210 40% 96.1%)",
+				"muted-foreground": "hsl(215.4 16.3% 46.9%)",
+				background: "hsl(0 0% 100%)",
+
 				abeg: {
 					button: {
-						10: '#1C8384',
-						20: '#005E5F',
+						10: "#1C8384",
+						20: "#005E5F",
 					},
 					teal: {
-						DEFAULT: '#268384',
-						10: '#2B908E',
+						DEFAULT: "#268384",
+						10: "#2B908E",
 					},
 					green: {
-						DEFAULT: '#324823',
-						20: '#7C946B',
-						30: '#C9DDBB',
-						40: '#F4FAF0',
-						50: '#007004',
-						60: '#006004',
+						DEFAULT: "#324823",
+						20: "#7C946B",
+						30: "#C9DDBB",
+						40: "#F4FAF0",
+						50: "#007004",
+						60: "#006004",
 					},
 					neutral: {
-						10: '#101E14',
-						20: '#344639',
-						30: '#4E5F53',
-						40: '#748178',
-						50: '#A1AAA4',
-						60: '#CDD6D0',
-						70: '#E3E8E5',
-						80: '#F4F5F5',
+						10: "#101E14",
+						20: "#344639",
+						30: "#4E5F53",
+						40: "#748178",
+						50: "#A1AAA4",
+						60: "#CDD6D0",
+						70: "#E3E8E5",
+						80: "#F4F5F5",
 					},
 					error: {
-						10: '#6A0101',
-						20: '#FC3131',
-						30: '#FEDCDC',
-						40: '#FFEBEB',
+						10: "#6A0101",
+						20: "#FC3131",
+						30: "#FEDCDC",
+						40: "#FFEBEB",
 					},
 				},
 			},
-			height: {
```

Comment:
abeg no delete this height. it's important

Reply:
Oh yeah! I'll revert that... Was testing a few things with it👍